src/Sulu/Bundle/MediaBundle/Resources/public/js/services/media-router.js

```diff
@@ -95,6 +96,14 @@ define([
                 }
             });
 
+            // show a single collection with files and upload (without locale)
+            routes.push({
+                route: mediaRoutes.editCollectionWithoutLocale,
+                callback: function(id) {
+                    MediaRouter.toCollection(id);
```

Comment:
It does exactly that, it navigates to the URL with the locale. The route is needed, because otherwise I'd have to hardcode a locale for the behat action "Given I am editing the collection `xy`"